perf/proxy_service_notifier.go

```diff
@@ -0,0 +1,104 @@
+package perf
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/evergreen-ci/cedar/model"
+	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+// PerformanceAnalysisProxyService is the interface for the Proxy Service.
```

Comment:
`...the client interface for interacting with a performance analysis proxy service.`